### DISCUSSION TITLE
client id and client secret

### DISCUSSION THREAD

**Discussion by Davidmax2023** [ip-tools/python-epo-ops-client#76] [Q&A] [answered]
when I test OPS API schema ,it require client id and client secret. can you tell me how to get this info?

**Comment by gsong** [comment, marked as answer]
https://developers.epo.org/user/register

**Reply by Davidmax2023** [reply]
Thank you, it's ok now. i have another question, there are many endpoint ,each one requires different parameter.is there any naming specification about how to set these search term?

**Reply by amotl** [reply]
Hi David. We will do something about documentation, thanks!

- GH-139
